src/site/stages/build/plugins/create-react-pages.js

```diff
@@ -1,30 +1,27 @@
 /* eslint-disable no-param-reassign, no-continue, no-console */
 const path = require('path');
-const manifestHelpers = require('../../../../../config/manifest-helpers');
+const appRegistry = require('../../../../applications/registry.json');
```

Comment:
Should we also go ahead and update other uses of `manifestHelpers.getAppManifests()` to pull from the registry instead?

Reply:
![image](https://user-images.githubusercontent.com/12970166/80538570-d01f5e00-895a-11ea-9a5f-0a719e8cc1f6.png)
I don't think so. I think it makes sense for these ones to continue to use the manifest files. The `registry.json` will be leaving `vets-website` with the content build, but we'll still need these pieces.